launcher/src/components/UI/node-header/IconsNav.vue

```diff
@@ -34,49 +21,35 @@
     </div>
 
     <router-link to="/setting" class="icon-btn">
-      <div
-        @mouseover="showSettingText = true"
-        @mouseleave="showSettingText = false"
-      >
+      <div>
         <img alt="Login" src="/img/icon/header-icons/setting4.png" />
-        <span class="setting-text" v-if="showSettingText">Setting</span>
       </div>
     </router-link>
 
-
-    <div
-      class="icon-btn"
-      @mouseover="showExitText = true"
-      @mouseleave="showExitText = false"
-    >
+    <div class="icon-btn">
       <img alt="Login" src="/img/icon/header-icons/exit9.png" />
     </div>
-    <update-modal
-      @remove-modal="removeModalHandler"
+    <update-panel
+      v-if="displayUpdatePanel"
       @update-confirm="updateConfirmationHandler"
-      v-if="showUpdateModal"
-    ></update-modal>
-    <update-waiting v-if="updateWaitingModal"></update-waiting>
+      @run-update="runUpdate"
```

Comment:
this won't work runUpdate needs an item from `newUpdates` to work.
i would move the functions to `UpdatePanel.vue`.